src/Sentry.Extensions.Logging/Sentry.Extensions.Logging.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net6.0;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net8.0;net6.0;netstandard2.0</TargetFrameworks>
```

Comment:
oh big save! we need this one. 

Best do a review of all projects before we close 4.0

Reply:
Currently the following projects **do not** target `net8.0`:

- Sentry.Android.AssemblyReader.csproj
- Sentry.AspNet.csproj
- Sentry.AspNetCore.Grpc.csproj
- Sentry.Bindings.Android.csproj
- Sentry.Bindings.Cocoa.csproj
- Sentry.DiagnosticSource.csproj
- Sentry.EntityFramework.csproj
- Sentry.Google.Cloud.Functions.csproj
- Sentry.Log4Net.csproj
- Sentry.Maui.csproj

I think we've had this conversation before... some of those are legacy (e.g. AspNet, Log4Net, and EntityFramework). Anything that needs to support AOT will need a net8.0 target though - including the bindings presumably? 

I'll create a new issue from this comment for the 4.0.0 milestone.